_schema.graphql

```diff
@@ -9624,15 +9620,30 @@ type PopularArtists {
   artists: [Artist]
 }
 
+type Price {
+  # An amount of money expressed in cents (minor units).
+  cents: Float
+
+  # The ISO-4217 alphabetic currency code, as per https://en.wikipedia.org/wiki/ISO_4217
+  currency: String
+
+  # A pre-formatted price.
+  display: String
+
+  # An amount of money expressed in dollars (major units).
+  dollars: Float
```

Comment:
Per this working doc: https://www.notion.so/artsy/Currency-Best-Practices-95081d370e664509928509e3dbd74c81#8412fa3a336447119463590f950f5c32

Should we instead call these `majorUnits` and `minorUnits` (vs. `dollars` and `cents`)?

Reply:
I'm 👍 with this

Reply:
should it be `majorUnits` or `major`?

Reply:
`majorUnit` and `minorUnit` imo

Reply:
I leaned into the proposal from the currency taskforce.

Reply:
Not sure if you got a response from the group but since we're discussing naming... 😁 -- `majorUnit` and `minorUnit` do feel better to me than `major` or `minor`.

(`major`/`minor` being adjectives and `majorUnit`/`minorUnit` being nouns since we're describing a _thing_.)

Reply:
@alloy I never did answer you question 😄. I think `majorUnits` is the same sort of pluralization that `dollars` gets. I'm not a language expert,  but it seems like `majorUnit` would be returning the _type_ of major currency where `majorUnits` seems to imply multiple of something. 

`Price` in `majorUnit` vs `Price` in `majorUnits`?

Reply:
According to the doc (and I think I agree), `major` and `minor` are already unambiguous within a "price" context.

If I read "units" in any of these names, I would expect to see the name of the major or minor unit in that field, like "dollars" or "cents."

(Minor fields should _definitely_ be integers... that's the whole point.)

Reply:
I did a [poll](https://artsy.slack.com/archives/CHP3F5NGM/p1571845579006600) on this in #currency-task-force and it leaned more towards `major` than `majorUnits`. For the time being I'm going to move forward with what's implemented here, but I think it's valid that we can revisit this before spreading the pattern.